ic-os/components/networking/nftables/hostos/nftables.template

```diff
@@ -175,6 +175,14 @@ table ip6 filter {
     accept
   }
 
+  set local_prefix {
+    type ipv6_addr
+    flags interval
+    elements =  {
+      {{ ipv6_prefix }}
```

Comment:
The config is slightly different but functionally the same for HostOS, to match how the rest of the rules are written.